OpenSim/Common/SmoothSegmentedFunction.h

```diff
@@ -24,9 +24,15 @@
  * -------------------------------------------------------------------------- */
 #include "osimCommonDLL.h"
 #include "SegmentedQuinticBezierToolkit.h"
+#include <memory>
 
 namespace OpenSim { 
 
+    /**
+    Class containing the data used by SmoothSegmentedFuncion.
+    */
+    class SmoothSegmentedFunctionData;
```

Comment:
Based on the CI failure for Windows, I think this forward declaration should be `struct SmoothSegmentedFunctionData`.

Reply:
Right! I think that would fix it, I pushed the change.